go-controller/pkg/node/gateway_init_linux_test.go

```diff
@@ -1857,7 +1857,6 @@ var _ = Describe("Gateway unit tests", func() {
 				gwIPs := []net.IP{config.Gateway.MasqueradeIPs.V4DummyNextHopMasqueradeIP}
 				config.Gateway.Interface = dummyBridgeName
 				config.Gateway.Mode = config.GatewayModeLocal
-				config.Gateway.AllowNoUplink = true
```

Comment:
self-note: there is another UT testing with allowNoUplink to true so we do have coverage.